.circleci/continue_config.yml

```diff
@@ -69,6 +69,7 @@ commands:
           command: |
             cd <<parameters.plugin>>
             pip install git+https://github.com/kedro-org/kedro@main
+            pip install .
```

Comment:
This affects all plugins - they were migrated in a similar way: `test_requirements.txt` does not pull `requirements.txt` anymore because it doesn't exist, so installing the plugin itself is necessary.